pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+### Purpose
+_High level description of what the new or modified functionality is._
+
+### Impact
+_What products/areas are potentially impacted by these changes?_
+- [ ] Publish
+- [ ] Analyze
+- [ ] Reply
+- [ ] Mobile
+- [ ] Data Tracking
+- [ ] Global Admin
+- [ ] Buffer Admin
+- [ ] Reply HQ
+- [ ] Security (anything involving access tokens, exposing APIs, etc.)
+- [ ] Other (please explain)
+
+### Related Links 
+_JIRA tickets, other PRs, Paper docs, etc._
+
+### Priority
+_If you're someone outside of the Core team, please describe the priority of this work for your product team.  For example is it blocking some other high priority work, is it a critical bug fix, is it needed for a scheduled release in a week, etc._
+
+### Notes
+
+### Review
+
+#### Staging Deployment
+
+---
+
+### Instructions for Folks Outside of Core Creating PRs
+- Please add at least one Core engineer as a reviewer
+- Be sure to fill in the Purpose, Impact, and Priority sections above
+- Ping the Core team in Slack (you can use @-core in #collab-analyze-core, #collab-reply-core, or #prod-core depending on what team you're in) to give a heads up that the PR is ready for review.  If the change is urgent/critical, include this context.
+- Wait for a Core engineer to get back to you.  If the scope of the changes feels small and the risk feels low, the Core team may give you the green light to go ahead and merge the changes now and we'll circle back with the review afterwards.  If the changes feel more risky or have a broader impact, we may ask that you wait to merge the changes until someone from Core has had a chance to review the changes.
+- After your changes have been merged and deployed, please monitor Bugsnag to see if any new errors are reported.
```

Comment:
Heya @philippemiguet!  I wasn't sure which Bugsnag dashboard(s) to link, so I've not included a link.  Please feel free to add a link before merging.